rs/nns/governance/src/gen/ic_nns_governance.pb.v1.rs

```diff
@@ -4707,6 +4707,10 @@ pub enum NnsFunction {
     DeployHostosToSomeNodes = 51,
     /// The proposal requests a subnet rental.
     SubnetRentalRequest = 52,
+    /// Instruct the migration canister to not accept any more migration requests.
+    PauseMigrations = 53,
```

Comment:
Can we make this "PauseCanisterMigrations"? Governance may interact in the future with some other canister that handles some other kind of migration.

Similarly, UnpauseCanisterMigrations?